cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1798,23 +1798,22 @@ pub(crate) fn define(
     }
 
     // SIMD extractlane
-    let mut x86_pextr_mapping: HashMap<u64, (&'static [u8], Option<SettingPredicateNumber>)> =
-        HashMap::new();
-    x86_pextr_mapping.insert(8, (&PEXTRB, Some(use_sse41_simd)));
-    x86_pextr_mapping.insert(16, (&PEXTRW_SSE2, None));
-    x86_pextr_mapping.insert(32, (&PEXTR, Some(use_sse41_simd)));
-    x86_pextr_mapping.insert(64, (&PEXTR, Some(use_sse41_simd)));
+    let mut x86_pextr_mapping: HashMap<u64, &'static [u8]> = HashMap::new();
+    x86_pextr_mapping.insert(8, &PEXTRB);
+    x86_pextr_mapping.insert(16, &PEXTRW);
```

Comment:
(meta comment about this commit's message: using `#1` in the message will mention unrelated issues. Not a big deal of course.)

Also, would we want to still have the shorter encoding in the future, when we want to optimize for size (and don't have access to SSE4.1)?

Reply:
Yes, we will need the shorter encoding to finish #1015 but for now I thought it best to get rid of it; it just caused confusion when investigating this bug because it is a valid opcode but requires a completely different recipe. It will be easy to re-add in #1015 when we get to that.